website/docs/d/vra_region_enumeration_vmc.html.markdown

```diff
@@ -0,0 +1,38 @@
+---layout: "vra"
+page_title: "VMware vRealize Automation: vra_region_enumeration_vmc"
+description: |-
+  Provides a data lookup for vra_region_enumeration_vmc.
+---
+
+# Data Source: vra_region_enumeration_vmc
+## Example Usages
+
+This is an example of how to read a region enumeration data source for VMC cloud account.
+
+**Region enumeration VMC data source by its id:**
+```hcl
+data "vra_region_enumeration_vmc" "this" {
+  id = vra_region_enumeration_vmc.this.id
+}
+```
+## Argument Reference
+
+* `id` - (Optional) The id of the region enumeration for VMC account.
+
+## Attribute Reference
+
+* `accept_self_signed_cert` - Accept self signed certificate when connecting to vSphere. Example: false
+
+* `api_token` - Host name for the cloud account endpoint. Example: dc1-lnd.mycompany.com
+
+* `dc_id` - Identifier of a data collector vm deployed in the on premise infrastructure. Refer to the data-collector API to create or list data collectors.
+
+* `vcenter_hostname` - The IP address or FQDN of the vCenter Server in the specified SDDC. The cloud proxy belongs on this vCenter.
+
+* `vcenter_password` - Password for the user used to authenticate with the cloud Account
+
+* `sddc_name` - Identifier of the on-premise SDDC to be used by this cloud account.
+
+* `vcenter_username` - vCenter user name for the specified SDDC.The specified user requires CloudAdmin credentials. The user does not require CloudGlobalAdmin credentials.
```

Comment:
Same comment as above. Correct the example. Organize arguments and attributes correctly.